management/server/user.go

```diff
@@ -95,6 +98,7 @@ func (u *User) Copy() *User {
 		Id:         u.Id,
 		Role:       u.Role,
 		AutoGroups: autoGroups,
+		PATs:       u.PATs,
```

Comment:
I think that there should be an array copy. Not just a reference. Otherwise the the old user will keep the reference